src/e2e.spec.js

```diff
@@ -311,6 +311,24 @@ describe('e2e', () => {
           { node: { name: 'Foo' } },
           { node: { name: 'Mother' } }
         ]);
+
+        result = await graphql(schema, `{
+          viewer {
+            users(orderBy: NAME_DESC) {
+              edges {
+                node {
+                  name
+                }
+              }
+            }
+          }
+       }`);
+
+        expect(result.data.viewer.users.edges).to.be.eql([
+          { node: { name: 'Mother' } },
+          { node: { name: 'Foo' } },
+          { node: { name: 'Bar' } }
+        ]);
       });
 
       it('should return results in descending order', async function Test() {
```

Comment:
I'd rename this one in something like "should be able to limit the ordered results"